nautobot/docs/user-guide/platform-functionality/approval-workflow.md

```diff
@@ -1 +1,345 @@
+- Ensuring multiple stakeholders approve large-scale bulk edits.
+- Mandating peer review for scheduled jobs that affect multiple systems.
+
+## Model Reference
+
+```mermaid
+---
+title: Approval Workflow Entity Relationship Diagram
+---
+erDiagram
+    "extras.ApprovalWorkflowDefinition" {
+        string name
+        ContentType model_content_type FK
+        json model_constraints
+        int priority
+    }
+    "extras.ApprovalWorkflowStageDefinition" {
+        ApprovalWorkflowDefinition approval_workflow_definition FK
+        int weight
+        string name
+        int min_approvers
+        string denial_message
+        Group approver_group FK
+    }
+    "extras.ApprovalWorkflow" {
+        ApprovalWorkflowDefinition approval_workflow_definition FK
+        GenericForeignKey object_under_review FK
+        ContentType object_under_review_content_type FK
+        uuid object_under_review_object_id
+        choices current_state
+        datetime decision_date
+        User user FK
+        string user_name
+    }
+    "extras.ApprovalWorkflowStage" {
+        ApprovalWorkflow approval_workflow FK
+        ApprovalWorkflowStageDefinition approval_workflow_stage_definition FK
+        choices state
+        datetime decision_date
+    }
+    "extras.ApprovalWorkflowStageResponse" {
+        ApprovalWorkflowStage approval_workflow_stage FK
+        User user FK
+        string comments
+        choices state
+    }
+
+    "contenttypes.ContentType"[ContentType] {}
+    "users.User"[User] {}
+    "users.Group"[Group] {}
+    "models.GenericForeignKey"[GenericForeignKey] {}
+
+    "extras.ApprovalWorkflowDefinition" }o--|| "contenttypes.ContentType": "applies to model"
+    "extras.ApprovalWorkflowDefinition" ||--o{ "extras.ApprovalWorkflowStageDefinition": "defines stages"
+    "extras.ApprovalWorkflowStageDefinition" }o--|| "users.Group": "approver group"
+    "extras.ApprovalWorkflowDefinition" ||--o{ "extras.ApprovalWorkflow": "creates instances"
+    "extras.ApprovalWorkflow" }o--|| "models.GenericForeignKey": "object under review"
+    "extras.ApprovalWorkflow" ||--o{ "extras.ApprovalWorkflowStage": "has stages"
+    "extras.ApprovalWorkflowStage" ||--o{ "extras.ApprovalWorkflowStageResponse": "has responses"
+    "extras.ApprovalWorkflowStageResponse" }o--|| "users.User": "submitted by"
+```
+
+### ApprovalWorkflowDefinition
+
+The template for a workflow, specifying which model(s) it applies to, any constraints, and the ordered list of stages. Represents a reusable definition of an approval workflow.
+
+#### Constraints
+
+Constraints define when a workflow definition should apply to a particular model instance.
+
+- Constraints are stored as a dictionary of **field lookups → values** (e.g. `{"status": "active"}`).
+- Any valid Django ORM lookup can be used (e.g. `{"name__in": ["A", "B"]}`, `{"status__icontains": "active"}`, `{"priority__gte": 10}`), since constraints are passed directly into `.filter(**constraints)`.
```

Comment:
`ScheduledJob` doesn't have a `status` or `priority` field, does it? Let's make sure the examples here are valid for the models that currently support workflows.